### PR TITLE
Provide console errors

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

File: src/summary.rs [hunk 1]
```diff
@@ -45,9 +51,67 @@ impl Summary {
...
+            for issue in self.issues.iter() {
+                let input = &issue.input;
+
+                let message: Option<String> = match &issue.kind {
+                    IssueKind::Disallowed(disallowed) => {
+                        if self.flake_check_config.check_supported {
+                            let reference = &disallowed.reference;
+                            Some(format!(
+                            "the `{input}` input uses the non-supported Git branch `{reference}` for Nixpkgs"
+                        ))
```
Comment (cole-h) [inline src/summary.rs:80]: Rustfmt chokes on these expressions with long lines, so you'll sadly need to hand-format them :(
```suggestion
                            Some(format!(
                                "the `{input}` input uses the non-supported Git branch `{reference}` for Nixpkgs"
                            ))
```

File: src/summary.rs [hunk 2]
```diff
@@ -74,7 +138,7 @@ impl Summary {
 
         let summary_txt = handlebars.render("summary.txt", &self.data)?;
 
-        std::io::stdout().write_all(summary_txt.as_bytes())?;
+        println!("{}", summary_txt);
```
Comment (cole-h) [inline src/summary.rs:141]: If you still want the error handling, you could use 
```suggestion
        writeln!(std::io::stdout(), "{}", summary_txt)?;
```

:D

(Not necessary though)

**Review by cole-h — APPROVED** [review]
LGTM though.

File: src/summary.rs (hunk 2 above)
Comment (lucperkins) [reply src/summary.rs:141]: Haha, I think `println` is fine 😄

**Pull request merged**